src/shell/components/favicon/favicon.js

```diff
@@ -271,20 +276,21 @@ export default connect((state) => {
         <ModalFooter>
           <ButtonGroup className={styles.Actions}>
```

Comment:
Do the material buttons play well with the zesty button group?

Reply:
They play nice but are unnecessary so I removed them up. I also swapped cancel and save location to follow material design primary secondary button location

![Screen Shot 2022-05-26 at 1 12 22 PM](https://user-images.githubusercontent.com/22800749/170572161-ed895216-a449-48c8-ad43-3d34d8e6f6f3.png)